plugins/UserLanguage/API.php

```diff
@@ -30,7 +30,6 @@ protected function getDataTable($name, $idSite, $period, $date, $segment)
         Piwik::checkUserHasViewAccess($idSite);
         $archive = Archive::build($idSite, $period, $date, $segment);
         $dataTable = $archive->getDataTable($name);
-        $dataTable->filter('Sort', array(Metrics::INDEX_NB_VISITS));
```

Comment:
No longer needed since we do now always sort in GenericFilter. So we do no longer sort the entire table twice